database/migrations/Guild/2024_06_11_215748_peoples_employment_history.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class () extends Migration {
+    /**
+     * Run the migrations.
+     */
+    public function up(): void
+    {
+        Schema::create('peoples_employment_history', function (Blueprint $table) {
+            $table->id();
+            $table->integer('peoples_id')->unsigned();
+            $table->integer('apps_id')->unsigned();
+            $table->string('position');
+            $table->decimal('income', 10, 2)->nullable();
+            $table->date('start_date');
+            $table->date('end_date')->nullable();
+            $table->integer('status')->default(0);
+            $table->string('income_type')->nullable();
+            $table->string('company_employer_name');
```

Comment:
@FredPeal change the name to company_name , we already are on the employment history table no need to add employer